lib/mutant/reporter/cli/printer.rb

```diff
@@ -258,7 +258,7 @@ def run
           # @api private
           #
           def coverage_percent
-            (coverage * 100).to_f
+            coverage * 100
```

Comment:
@mbj if you want I could extract this into it's own commit, since the extra `#to_f` is not needed in either case.